rebar.lock

```diff
@@ -1,30 +1,39 @@
-{"1.1.0",
-[{<<"certifi">>,{pkg,<<"certifi">>,<<"2.5.1">>},2},
+{"1.2.0",
```

Comment:
You're increasing the version of rebar3 used to build, is that intentional?

Reply:
Good catch, that was not intentional. I have fixed it.